styles/collection-listing.scss

```diff
@@ -58,13 +58,15 @@
         flex-direction: column;
         gap: 6px;
         position: absolute;
-        top: 50%;
-        left: 50%;
-        transform: translate(-50%, -50%);
         text-align: center;
-        margin-bottom: 24px;
         color: #fff;
-        width: 90%;
+        width: 100%;
+        height: 100%;
+        top: 0;
+        left: 0;
+        justify-content: center;
+        align-items: center;
+        background-color: rgba(0, 0, 0, 0.14);
```

Comment:
I believe this overlay color should come from the settings, user should be able to add the overlay color :)